lib/ui/auth_flow/create_profile_screen.dart

```diff
@@ -27,75 +33,133 @@ class _CreateProfileScreenState extends State<CreateProfileScreen> {
       resizeToAvoidBottomInset: true,
       body: SafeArea(
         child: SingleChildScrollView(
-          padding: const EdgeInsets.fromLTRB(24, 32, 24, 32),
+          padding: const EdgeInsets.fromLTRB(24, 32, 24, 0),
           child: Column(
             crossAxisAlignment: CrossAxisAlignment.center,
             children: [
-              const Center(
-                child: Text(
-                  'Set up your profile',
-                  style: TextStyle(fontSize: 22, fontWeight: FontWeight.bold),
+              const Text(
+                'Setup Your Profile',
+                textAlign: TextAlign.center,
+                style: TextStyle(
+                  fontSize: 30,
+                  fontWeight: FontWeight.w500,
+                  color: AppColors.glitch800,
                 ),
               ),
               const SizedBox(height: 32),
-              const CircleAvatar(backgroundColor: AppColors.black, radius: 40),
+
+              Stack(
+                alignment: Alignment.bottomRight,
+                children: [
+                  const CircleAvatar(
+                    radius: 50,
+                    backgroundImage: AssetImage('assets/pngs/avatar_placeholder.png'),
+                  ),
+                  Container(
+                    decoration: const BoxDecoration(
+                      color: Colors.white,
+                      shape: BoxShape.circle,
+                    ),
+                    padding: const EdgeInsets.all(4),
+                    child: const Icon(Icons.edit, size: 18),
+                  ),
+                ],
+              ),
               const SizedBox(height: 12),
+
               const Text(
-                'Upload photo',
-                style: TextStyle(fontSize: 16, fontWeight: FontWeight.w500),
+                'Upload Avatar',
+                style: TextStyle(
+                  fontSize: 16,
+                  fontWeight: FontWeight.w600,
+                  color: AppColors.glitch950,
+                ),
               ),
               const SizedBox(height: 32),
+
               const Align(
                 alignment: Alignment.centerLeft,
                 child: Text(
-                  'Choose a username',
-                  style: TextStyle(fontWeight: FontWeight.bold),
+                  'Choose a Name',
+                  style: TextStyle(
+                    fontWeight: FontWeight.w600,
+                    fontSize: 14,
+                    color: Colors.black,
+                  ),
                 ),
               ),
               const SizedBox(height: 8),
               TextField(
```

Comment:
We have a reusable customTextField widget for consistency. Could you replace this and other instances too?